packages/router/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@remix-run/router",
-  "version": "0.1.0",
+  "version": "0.2.0-pre.2",
```

Comment:
I wonder if we should move this to `1.0.0-pre.N` - I assume when `6.4.0` becomes stable we'll release the `1.0.0` of this package